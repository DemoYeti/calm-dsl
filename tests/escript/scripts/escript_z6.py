# python3;failure
pnt
