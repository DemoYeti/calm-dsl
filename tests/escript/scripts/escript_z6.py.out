Error in script: name 'pnt' is not defined
